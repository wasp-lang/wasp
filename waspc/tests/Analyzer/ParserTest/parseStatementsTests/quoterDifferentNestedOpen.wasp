test DifferentNestedOpen {=foo {=bar foo=}
