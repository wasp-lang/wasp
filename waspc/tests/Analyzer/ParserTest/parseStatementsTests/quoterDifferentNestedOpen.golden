(AST
  (Decl@1:1-42 type=test name=DifferentNestedOpen
    (Quoter@1:26-42 tag=foo
      {= {=bar =}
    )
  )
)
